src/app/components/home/home.component.ts

```diff
@@ -96,9 +107,24 @@ export class HomeComponent implements OnInit {
     this.desktopConfig = Constants.DESKTOP_CONFIG
     this.tabletConfig = Constants.TABLET_CONFIG
     this.mobileConfig = Constants.MOBILE_CONFIG
-     this.recommendedProductList()
-     this.recommendedSet()
+    // recommended prod
+    this.recommendedProductList()
+     //frequently bought
+     this.getFrequentlyBought().then(data => this.frequentlyBoughtSet(data))
+     //frequently bought by me
+     this.logInSession().then(data => {
+       if(data) {
+        this.getFrequentlyBoughtByMe().then(data => {
+          if(data === []){
```

Comment:
you can make use of the following
```suggestion
          if(data.length > 5){
```

Reply:
@reddy-s length property is not being recognised. And lets not restrict it to products more than 5